README.md

```diff
@@ -39,6 +39,8 @@ After creating your app, you'll need to create a new trigger that will start the
 slack triggers create --trigger-def ./trigger.ts
```

Comment:
plz update to say "trigger create",  the singular form

Reply:
Fixed!